core/mapreduce/src/main/java/mil/nga/giat/geowave/mapreduce/input/GeoWaveInputFormat.java

```diff
@@ -136,7 +136,7 @@ public static DataStatisticsStore getJobContextDataStatisticsStore(
 				context);
 	}
 
-	public static void addIndex(
+	public static void setIndex(
```

Comment:
Why change it to `setIndex`?  It looks like it still calls `addIndex` under the hood and it looks like you can still have multiple indices?  Am I missing something?